mm/kfence/core.c

```diff
@@ -414,6 +414,10 @@ static bool __init kfence_initialize_pool(void)
 		if (!i || (i % 2))
 			continue;
 
+		/* Verify we do not have a compound head page. */
```

Comment:
According to your own reply to Jann, "In slub.c, __GFP_COMP is passed to alloc_pages(), which causes them to
have a compound head I believe."
How come we don't have a compound head here then?

Reply:
Because we do not pass __GFP_COMP to alloc_pages() for our KFENCE pool.

Reply:
I see, I've misunderstod that comment.